### PR TITLE
2397 - working on review page

### PR DESCRIPTION
**Pull request by codyseibert**
- this PR refactors the request access view to match the review external document uplade page

Screen Shot 2019-10-02 at 11 19 49 AM

### CODE REVIEW

File: web-client/src/views/RequestAccess/RequestAccessReview.jsx [hunk 1]
```diff
@@ -2,76 +2,258 @@ import { Button } from '../../ustc-ui/Button/Button';
...
-              formCancelToggleCancelSequence();
-            }}
-          >
-            Cancel
-          </Button>
+            {requestAccessHelper.showPartiesRepresenting && (
+              <div className="tablet:grid-col-5 margin-bottom-4">
+                <div className="card height-full margin-bottom-0">
+                  <div className="content-wrapper">
+                    <h3 className="underlined">Parties You’re Repreesenting</h3>
```
Comment (kkoskelin) [inline web-client/src/views/RequestAccess/RequestAccessReview.jsx]: typo `Repreesenting`

**Review by kkoskelin — CHANGES_REQUESTED** [review]
One typo that needs to be fixed.  Also, if you're able to take screenshots of the things that were altered so UX can see them, it might be helpful.  Several folks have worked on altering the capitalization of various strings in the app, and I'm not 100% whether all the new strings here are conforming to the new rules (nor am I certain they're *not*).

Nice!

File: web-client/src/views/RequestAccess/RequestAccessReview.jsx (hunk 1 above)
Comment (matthopson) [inline web-client/src/views/RequestAccess/RequestAccessReview.jsx]: Typo "Representing"

**Review by kkoskelin — APPROVED** [review]

**Review by matthopson — APPROVED** [review]

File: web-client/src/views/RequestAccess/RequestAccessReview.jsx [hunk 2]
```diff
@@ -2,76 +2,258 @@ import { Button } from '../../ustc-ui/Button/Button';
 import { FileUploadErrorModal } from '../FileUploadErrorModal';
 import { FileUploadStatusModal } from '../FileUploadStatusModal';
 import { Focus } from '../../ustc-ui/Focus/Focus';
-import { PartiesRepresentingReadOnly } from './PartiesRepresentingReadOnly';
-import { RequestAccessDocumentReadOnly } from './RequestAccessDocumentReadOnly';
```
Comment (rachaelparris) [inline web-client/src/views/RequestAccess/RequestAccessReview.jsx:6]: Could you delete these 2 files if they're not being used anymore?

**Review by rachaelparris — APPROVED** [review]

File: web-client/src/views/RequestAccess/RequestAccessReview.jsx (hunk 2 above)
Comment (matthopson) [reply web-client/src/views/RequestAccess/RequestAccessReview.jsx:6]: @rachaelparris These have been removed. Good catch

**Pull request merged**